pymathics/natlang/translation.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+
+"""
+Language translation
+
+
+"""
+
+# This is under  Text Normalization in WR. But also in Natural Language Processing,
+# and Linguistic Data. I put here because is the only module that uses langid and pycountry
+# modules.
+#
+# TODO: WordTranslation, TextTranslation
+
+from typing import Union
+
+import langid  # see https://github.com/saffsd/langid.py
+import pycountry
+from mathics.builtin.base import Builtin
+from mathics.core.atoms import String
+from mathics.core.evaluation import Evaluation
+from mathics.core.symbols import Symbol
+from mathics.core.systemsymbols import SymbolFailed
+
+
```

Comment:
Here is where you note the missing WordDataFrequencyData (and possibly others in that section like PartOfSpeech, Transliterate, etc.

Reply:
`WordFrequencyData` seems to go with `linguistic_data.py`

Reply:
Ok I see that there is `linguistic_data.py`  - great! Maybe some of the others?